shesha-reactjs/src/components/formDesigner/components/sizableColumns/__tests__/index.test.tsx

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
```

Comment:
There's no point of this. Please remove this unit shallow unit test

Reply:
Was added as to follow the standard of all components